.github/workflows/static.yml

```diff
@@ -0,0 +1,29 @@
+name: Check collectstatic
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.12.6
```

Comment:
If you put this in a `.python-version`, you can remove it from here:

```suggestion
```

And here: https://github.com/python/pythondotorg/blob/019c0629fd98c7ed4005387eb4ece5182995b14f/.github/workflows/ci.yml#L33-L34

https://github.com/actions/setup-python#basic-usage

Reply:
without _something_ set there's a warning

```
Warning: Neither 'python-version' nor 'python-version-file' inputs were supplied. Attempting to find '.python-version' file.
```

I'll explicitly set it.